src/components/Navbar/navbar.css

```
@@ -78,13 +86,13 @@ header{
     align-items: center;
     width: 100%;
     height: fit-content;
-    padding: 10px 100px;
-    position: fixed;
-    box-shadow: 1px 0 10px rgb(178, 178, 178);
+    padding: 10px 30px;
```

```suggestion
    padding: 16px 30px;
```
Μια τιμή για το vertical padding κοντά στα 16px νομίζω είναι καλή.